indico/modules/rb/client/js/modules/admin/LocationPage.jsx

```diff
@@ -199,6 +213,48 @@ class LocationPage extends React.PureComponent {
           />
         )}
       </Field>
+      <Form.Field>
+        <h5>
+          <Translate>Protection Mode</Translate>
```

Comment:
I'm a bit surprised to see this, since IIRC this was initially supposed to be just about being able to define location-based permissions, but not about restricting visibility (something we do not have anywhere else in the RB module).

Especially since this would need to be taken into account everywhere, not just for rooms but also to filter bookings etc.

It's the kind of thing I'd rather see in a separate PR at a later point if this functionality is necessary...

Reply:
I added it already so that they use the same db migration script since we plan later to add the bookers at the location level then bookers/prebooker defined at the location will be able to book/prebook protected rooms in that location. But anyway it is not yet being used so can be removed from this PR.